### ISSUE TITLE
tune deterministic resnet-50 to 76.3%?

### ISSUE THREAD

**Issue by dustinvtran** [google/uncertainty-baselines#58]
> @dustinvtran 
> > [Facebook's scaling paper](https://research.fb.com/wp-content/uploads/2017/06/imagenet1kin1h5.pdf) suggests they consistently get 76.4% with base learning rate=0.1 and total_batch_size (kn) = 256, and 76.3% with total_batch_size (kn)=8k. Our accuracies are hovering around 76.0-76.3%, based on the official tpu resnet50 keras codebase. Should double check whether the official codebase is meeting this target and ultimately how we can meet it.

https://github.com/google/edward2/issues/143
